speedrun-3.html

```diff
@@ -4742,6 +4730,19 @@ <h3>Made by Ben Songster (MeemawHustlin) with help from the PRCLive community.</
 </footer>
 <!-- END FOOTER-->
 </article><!-- main end-->
+
+<div class="resizer" id="dragMe"></div>
+
+<div class="sidePanel" id="sidebar">
+	<div id="sidebarContent" class="sidebarContent">
+		<div id="sidebarLinks">
+			<a href="./help.html" target="myframe">🛈</a>
+			<a href="./toc-3.html" target="myframe">📄</a>
+			<a href="./map.html?topbar=false" target="myframe" title="Show Map">🗺️</a>
```

Comment:
Eventually I want to incorporate fontawesome since the prclive home page already has it and we'll have access to it so we can use those in addition to emojis. Some emojis like the map actually look terrible on specific devices lol since each emoji actually has several ways they can appear on different devices but in general they are really good for our purposes.